src/adhocracy_core/adhocracy_core/workflows/__init__.py

```diff
@@ -0,0 +1,81 @@
+"""Finite state machines for resources."""
+from colander import Invalid
+
+from pyramid.interfaces import IRequest
+from pyramid.registry import Registry
+from substanced.workflow import ACLWorkflow
+from substanced.workflow import WorkflowError
+from substanced.workflow import IWorkflow
+from zope.interface import implementer
+
+from adhocracy_core.exceptions import ConfigurationError
+from adhocracy_core.interfaces import IAdhocracyWorkflow
+from adhocracy_core.workflows.schemas import create_workflow_meta_schema
+
+
+@implementer(IAdhocracyWorkflow)
+class AdhocracyACLWorkflow(ACLWorkflow):
+
+    """Workflow that sets the :term:`acl` when entering a State."""
+
+    def get_next_states(self, context, request: IRequest) -> list:
+        """Get states you can trigger a transition to."""
+        state = self.state_of(context)
+        transitions = self.get_transitions(context, request, from_state=state)
+        states = [t['to_state'] for t in transitions]
```

Comment:
if I understand correctly if two or more transitions lead to the same state, the state will appear multiple times in the returned list. Is this wanted?